src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -42,41 +51,36 @@ public static void main(String[] args) {
     Logger.setup();
     logSystemInfo();
 
-    // Determine if user intends to use the command-line interface, and gather args if so
-    boolean useCli = false;
-    List<String> argsCli = new ArrayList<>();
-    for (String arg : args) {
-      if (!useCli && arg.equals("-cli")) {
-        useCli = true;
-      } else if (useCli) {
-        argsCli.add(arg);
-      }
-    }
-
-    if (!useCli) {
-      // Launch the GUI
+    // Check if args contains string "--cli"
+    if (Arrays.stream(args).filter(arg -> arg.equals("--cli")).findAny().isEmpty()) {
+      // --cli not found. Launch the GUI
       launch(args);
     } else {
-      Logger.info("Tabulator is being used via the CLI.");
-      // Check for unexpected input
-      if (argsCli.size() == 0) {
-        Logger.severe(
-            """
-                No config file path provided on command line!
-                Please provide a path to the config file!
-                See README.md for more details.""");
-        System.exit(1);
-      } else if (argsCli.size() > 2) {
-        Logger.severe(
-            "Too many arguments! Max is 2 but got: %d\n" + "See README.md for more details.",
-            argsCli.size());
-        System.exit(2);
+      CommandLine cmd = parseArgsForCli(args);
+      String path = cmd.getOptionValue("cli");
+      String name = cmd.getOptionValue("name");
+      Boolean convertToCdf = cmd.hasOption("convert-to-cdf");
+
+      if (name == null || name.isEmpty()) {
+        // Prompt user for their name
+        System.out.println("Enter name(s) of operators for auditing purposes:");
```

Comment:
Can we provide a name of "Automated Test" any time we run a test somewhere? And, if so, would it simplify any of this logic?

Reply:
We can provide any name any time we test. If provided via cli argument, you won't be prompted here. Not sure if that's what you mean by simplify?

Reply:
Nevermind, I didn't realize that all this is just bypassed for our tests.